### PR TITLE
Port documentation from Wiki #331 and #332 and images from new adminUI

### LINKED ISSUE #331: Port documentation from Wiki: Reference field
Port old article from wiki and refresh them for sensenet 7. The new article should be an .md file in the /_docs folder. Old screenshots should be re-created on the new admin-ui if it is possible and placed into the /docs/images folder.
[admin.sensenet.com](https://admin.sensenet.com)
name: BuiltIn\admin
password: admin
repository url: https://dev.demo.sensenet.com

The .md file should have the following header:

```
---
title: %title%
source_url: %url of the file on github%
category: Development
version: v7.0.0
tags: %possible tags as an array%
---
```

(you can use [the docs here](https://github.com/SenseNet/sensenet.github.io/tree/master/_docs) as example)

[Reference field](http://wiki.sensenet.com/Reference_Field)

Please change the old wiki urls as if they were available on github (e.g. https://wiki.sensenet.com/content_templates sould be changed to /content-templates)

### LINKED ISSUE #332: Port documentation from Wiki: ShortText Field
Port old article from wiki and refresh them for sensenet 7. The new article should be an .md file in the /_docs folder. Old screenshots should be re-created on the new admin-ui if it is possible and placed into the /docs/images folder.
[admin.sensenet.com](https://admin.sensenet.com)
name: BuiltIn\admin
password: admin
repository url: https://dev.demo.sensenet.com

The .md file should have the following header:

```
---
title: %title%
source_url: %url of the file on github%
category: Development
version: v7.0.0
tags: %possible tags as an array%
---
```

(you can use [the docs here](https://github.com/SenseNet/sensenet.github.io/tree/master/_docs) as example)

[ShortText field](http://wiki.sensenet.com/ShortText_Field)

Please change the old wiki urls as if they were available on github (e.g. https://wiki.sensenet.com/content_templates sould be changed to /content-templates)

### PR DESCRIPTION
**Pull request by cyberShaw**
Fixes #331 
Fixes #332

### CODE REVIEW

File: _docs/shorttext-field.md [hunk 1]
```diff
@@ -63,11 +57,14 @@ Fully featured example:
...
 - at most 15 characters are allowed to be inserted
 - at least 3 characters are expected to be inserted
 - Field value is editable (not read-only)
-- Field can contain only letters and numbers
\ No newline at end of file
+- Field can contain only letters and numbers
+
+## Related Links
+- [Field](/docs/field)
+- [ShortText Field Control](/docs/shorttext-fieldcontrol)
```
Comment (herflis) [inline _docs/shorttext-field.md]: pls remove this part

File: _docs/reference-field.md [hunk 2]
```diff
@@ -79,6 +80,12 @@ Fully featured example:
...
 - content can only be referenced from under _/Root/System_ and _/Root/Sites_
-- by default _/Root/System/SystemPlugins_ and _/Root/System/Schema_ is referenced
\ No newline at end of file
+- by default _/Root/System/SystemPlugins_ and _/Root/System/Schema_ is referenced
+
+## Related Links
+- [Field](/docs/field)
+- [ReferenceGrid Field Control]()
+- [Node for developers](/docs/node-for-developers)
+- [ContentPicker]()
```
Comment (herflis) [inline _docs/reference-field.md]: pls remove this part

**Comment by herflis** [comment]
Hey @cyberShaw ,
thank you for your contribution! 🙏 I've made some minor suggestions, but otherwise it is ok. 🙂

File: _docs/reference-field.md (hunk 2 above)
Comment (cyberShaw) [reply _docs/reference-field.md]: I've removed the parts that you've requested in two new commits. Should I open a new PR?

Comment (herflis) [reply _docs/reference-field.md]: No, you do'nt have to create another pr. Thank you again for your help!

**Pull request merged**